packages/client/src/TRPCClientError.ts

```diff
@@ -10,6 +10,8 @@ import {
   type DefaultErrorShape,
 } from '@trpc/server/unstable-core-do-not-import';
 
+type inferErrorShape<TInferrable extends InferrableClientTypes> =
```

Comment:
looks unused?

Reply:
It's just moved in-line, it was only used here